src/data/zcl_abapgit_data_deserializer.clas.testclasses.abap

```diff
@@ -26,7 +26,6 @@ CLASS ltcl_test IMPLEMENTATION.
 
     li_config->add_config( ls_config ).
 
-* this does not change the database,
     li_cut->deserialize(
```

Comment:
will this unit test now do database changes?

Reply:
it certainly shouldn't. only `actualize` updates DB

Reply:
updated comment